src/main/resources/templates/addFacility.html

```diff
@@ -14,25 +14,25 @@
 
```

Comment:
I moved the syle.css file to the static folder inside of resources; it should not be kept inside of the templates folder because spring will not load its resources. However, if you add it to the static folder, spring will load it into the web context at the base URL of the application. The stylesheet was not accessible when running the application. I updated the paths accordingly. I would recommend moving it here if you want to use the style.css

Reply:
Agreed.  Good point.